TRAMS-API/Controllers/AcademiesController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using API.Mapping;
+using API.Models.D365;
+using API.Models.Response;
+using API.Repositories;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace API.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("[controller]")]
+    [ProducesResponseType(typeof(string), StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status500InternalServerError)]
+    public class AcademiesController : ControllerBase
+    {
+        private readonly AcademiesRepository _academiesRepository;
+        private readonly IMapper<GetAcademiesD365Model, GetAcademiesModel> _getAcademiesMapper;
+
+        public AcademiesController(AcademiesRepository academiesRepository,
```

Comment:
Suggestion: Break this out into an interface so that it can be mocked or switched for another implementation in the future.